python/bigquery_handler.py

```diff
@@ -0,0 +1,136 @@
+import gzip
+
+from google.cloud import bigquery
+import json
+import os
+import re
+
+from plbart_relevant_code import preprocess
+
+project_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__))).replace("\\", "/") + '/'
+count_in_method = 0
+count_end_method = 0
+all_methods_dictionary = {}
+
+
+def tokenize_bigquery_json():
+    global all_methods_dictionary
+    filepath = project_path + 'bigquery_files/bq-results-20220207-151512-biubj4ywzbd1.json'
+    # filepath = project_path + 'bigquery_files/test.json'
+    data = []
+    with open(filepath, "r+", encoding='utf8') as file:
+        line = file.readline()
+        while line:
+            data.append(json.loads(line))
+            line = file.readline()
+    for instance in data:
+        assert not instance['binary']
```

Comment:
I wanted to make sure there aren't any binary files (and there aren't). also, there're not supposed to be any, because in the BigQuery SQL I relied on text in the content section (import selenium lines), so I think I'll delete this line.